### PR TITLE
fix: find-slot little fix and tests fixes after Chrome was updated to…

### PR DESCRIPTION
**Pull request by adevnadia**
… the 53 version

### CODE REVIEW

File: src/util/find-slots.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
-import { shadowDomV0 } from './support';
+import { shadowDomV0, shadowDomV1 } from './support';
 import isSlotNode from './is-slot-node';
 
 export default function findSlots(root, slots = []) {
   const { childNodes } = root;
 
-  if (shadowDomV0) {
+  if (shadowDomV0 && !shadowDomV1) {
```
Comment (jpnelson) [inline src/util/find-slots.js:7]: when would it be v0 and also v1?

File: test/unit/slot/distribution.js [hunk 2]
```diff
@@ -145,8 +145,8 @@ describe('slot/distribution', () => {
     });
 
     it('for named slots', () => {
-      const host1 = document.createElement('host1');
-      const host2 = document.createElement('host2');
+      const host1 = document.createElement('host-1');
+      const host2 = document.createElement('host-2');
       const slot1 = document.createElement('slot');
       const slot2 = document.createElement('slot');
       const test = document.createElement('test');
```
Comment (jpnelson) [inline test/unit/slot/distribution.js:152]: if `host1` and `host2` need `-`'s, `test` would as well, right?

File: src/util/find-slots.js (hunk 1 above)
Comment (adevnadia) [reply src/util/find-slots.js:7]: in Chrome 53 ;)

File: test/unit/slot/distribution.js (hunk 2 above)
Comment (adevnadia) [reply test/unit/slot/distribution.js:152]: surprisingly no

File: test/unit/util/find-slots.js [hunk 3]
```diff
@@ -1,12 +1,11 @@
 import findSlots from '../../../src/util/find-slots';
-
+/* eslint-disable no-unused-expressions */
```
Comment (lukebatchelor) [inline test/unit/util/find-slots.js]: What is this for sorry?

**Comment by lukebatchelor** [comment]
LGTM

File: test/unit/util/find-slots.js (hunk 3 above)
Comment (adevnadia) [reply test/unit/util/find-slots.js]: You didn't see anything ;)

**Comment by adevnadia** [comment]
lgtm

**Pull request merged**